scripts/generate-launch-args.py

```diff
@@ -20,6 +21,7 @@
     'rhel8': 'SKIP_TESTTYPES_RHEL8',
     'rhel9': 'SKIP_TESTTYPES_RHEL9',
     'rhel10': 'SKIP_TESTTYPES_RHEL10',
+    'centos10': 'SKIP_TESTTYPES_CENTOS10',
 }
 
 RE_MASTER = re.compile('^main$')
```

Comment:
Don't we need also `RE_CENTOS10` and updating the `get_arguments_for_branch` function below?

Reply:
I don't expect we will have centos-10 GH branch.
Maybe we'll run centos10 platform tests (additionally to rhel10) on rhel-10 branch